parse.py

```diff
@@ -990,18 +1005,19 @@ def price_item(text):
 
                         print("[!] Not enough data to confidently price this item.")
                         if USE_GUI:
-                            gui.show_price(price, price_vals, time, True)
+                            priceInfo.add_price_info(price, price_vals, time, True)
```

Comment:
I personally liked when you had to explicitely call create_at_cursor() for every component instead of having it hidden inside this method